src/main/kotlin/step2/domain/Lotto.kt

```diff
@@ -0,0 +1,19 @@
+package step2.domain
+
+data class Lotto(var numbers: List<Int>) {
+    init {
+        require(numbers.size == 6) { "6자리의 번호가 필요합니다." }
+    }
+}
+
+fun List<Lotto>.getAnswerCountList(answer: Lotto): List<Int> {
+    return this.map { lotto ->
+        lotto.numbers.map {
+            var result = 0
+            if (answer.numbers.contains(it)) {
+                result++
+            }
+            result
+        }.count { it > 0 }
```

Comment:
```suggestion
        lotto.numbers.count { answer.numbers.contains(it) }
```
count 함수로 한번에 작성할 수 있습니다 🙂